### PR TITLE
Add ECS livestate store

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/livestatestore/ecs/store.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		With(zap.String("cloud-provider", cloudProvider))
+
+	return &Store{
+		logger: logger,
+	}
+}
+
+func (s *Store) Run(ctx context.Context) error {
+	s.logger.Info("start running ecs app state store")
+
```
Comment (nakabonne) [inline pkg/app/piped/livestatestore/ecs/store.go:48]: You look to add the actual implementation here right?

Comment (khanhtc1202) [reply pkg/app/piped/livestatestore/ecs/store.go:48]: Yeh, currently just add this thing to unify with other deployment kinds. Just released I missed it while implemented it in the past.

Comment (nakabonne) [reply pkg/app/piped/livestatestore/ecs/store.go:48]: Gotcha

**Pull request merged**